state/alicloud/tablestore/tablestore_test.go

```diff
@@ -0,0 +1,136 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package tablestore
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/agrea/ptr"
```

Comment:
@theonefx can you run `goimports` to fix this linter issue?

Reply:
no problem

Reply:
done